app/database_etl/owid_ingestion_handler/owid_data_handler.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd
+import datetime
+import os
+
+from app.database_etl.location_utils import get_alternative_names, get_country_code
+
+CSV_DIR = os.getenv('CSV_DIR')
+
+vaccination_df = pd.read_csv(CSV_DIR + "vaccinations.csv")[['date', 'iso_code', 'people_vaccinated_per_hundred', 'people_fully_vaccinated_per_hundred']]
+tests_df = pd.read_csv(CSV_DIR + "tests.csv")[['Entity', 'Date', 'ISO code', 'Cumulative total per thousand']]  # per thousand
+cases_df = pd.read_csv(CSV_DIR + "cases.csv")  # per million
+deaths_df = pd.read_csv(CSV_DIR + "deaths.csv")  # per million
+
+
+def _get_alt_name(country_name, df):
```

Comment:
nit could add parameter typing here too, was a nice touch in all the other functions